pandas/tests/indexing/test_chaining_and_caching.py

```diff
@@ -452,7 +452,8 @@ def test_detect_chained_assignment_undefined_column(
                 df.iloc[0:5]["group"] = "a"
         else:
             with pytest.raises(SettingWithCopyError, match=msg):
-                df.iloc[0:5]["group"] = "a"
+                with tm.raises_chained_assignment_error():
+                    df.iloc[0:5]["group"] = "a"
```

Comment:
These few test changes illustrate cases where you now get both SettingWithCopyWarning and the new chained assignment warning. 

But for example the one above doesn't actually change `df` (so nobody should be doing that right now), so let's not care about the double warning?

(in theory I could specifically _not_ raise the warning when the key is a single column name, because assigning to a column is never inplace and thus can never change behaviour)

Reply:
Nah let's just live with both warnings, that shouldn't be something that users are doing anyway as you said